client/src/livingdocs-component-app/app.js

```diff
@@ -131,6 +131,7 @@ export class App {
 
       if (tool) {
         item.conf.icon = tool.icon;
+        item.conf.hasDynamicDisplayOptions = tool.hasDynamicDisplayOptions;
```

Comment:
mmh, this is a little dangerous - as is the line before - as `item.conf` holds the data from couchdb for this item. So we potentially overwrite anything defined in the tools schema here as far as I understand.

Should we hold that conf on a separate property here, maybe `item.__toolConf`?

Reply:
Yes, I agree. I created an object called `item.toolConfig` which holdes both `icon` and `hasDynamicDisplayOptions` properties  -> https://github.com/nzzdev/Q-editor/pull/192/commits/a49b3756bef7f9113a3bcf5e8943b68a8770d0d4